index.js

```diff
@@ -617,6 +617,9 @@ function speedTest(options) {
 
   function gotServers(err, servers) {
     if (err || !servers) return nextServer(err);
+    if (servers.settings == undefined) {
+      return new Error('No server found, verify your proxy/network');
```

Comment:
If you do it like this, the callback never gets called. I think it should be:

```
return nextServer(new Error('No server found, verify your proxy/network'));
```

Reply:
ok i change that now